src/document-converter.ts

```diff
@@ -1284,3 +1336,43 @@ function domModuleCanBeInlined(domModule: parse5.ASTNode) {
 
   return true;
 }
+
+/**
+ * Produces an array of preferred aliases for all `JsExports` referenced by a
+ * set of `ImportReference`s, given a set of undesirable aliases.
+ */
+function generateAliasRequests(
```

Comment:
I'm a little concerned with the perf for this function, considering it's run for every script.

Reply:
(This function has moved into 'src/document-util.ts', but isn't changed.)